test/smoke/migrations_test.go

```diff
@@ -550,7 +550,7 @@ func TestMigrations(t *testing.T) {
 	if err != nil {
 		t.Fatal("failed to create db:", err)
 	}
-	defer db.Exec("drop database " + sqlutil.QuoteID(dbName))
+	defer func(db *sql.DB) { _, _ = db.Exec("drop database " + sqlutil.QuoteID(dbName)) }(db)
```

Comment:
No need to pass it in, we can reference it from the outer scope.
- we're not in a loop
- it doesn't get reassigned

```suggestion
	defer func() { _, _ = db.Exec("drop database " + sqlutil.QuoteID(dbName)) }()
```